HHS/Head-Start-TTADP
# [TTAHUB-2117] National Centers drop down

## Description of change

This change allows admins to assign users to national centers. Once a user has been assigned to a national center they should no longer be assignable to any other national center.

Right now the user list is populated with users that have **read/write training reports** (this might change later).

![image](https://github.com/HHS/Head-Start-TTADP/assets/84350609/ae6c84f3-237a-4987-92d8-031bc8ee8096)

## How to test

Under the admin menu click the 'National Centers' button. Test **creating, updating, and deleting** national centers and their associated users. Make sure that once a user is assigned to a national center they can no longer be assigned to any other centers. Deleting a center should "release" that user to be assigned again.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2117


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
